python_modules/dagit/dagit/schema/errors.py

```diff
@@ -292,6 +292,49 @@ class Meta:
         )
 
 
+class DauphinStartSubPlanExecutionSuccess(dauphin.ObjectType):
```

Comment:
this is a dumb nit, but I think `Subplan` throughout rather than `SubPlan` would be better -- it's a distinct entity